Update DHCP Ranges

FletcherS7 opened the pull request:
Updated DHCP ranges to match FIRST FMS config
https://docs.wpilib.org/en/stable/docs/networking/networking-introduction/networking-basics.html#ethernet-wireless

Team DHCP range is now 10.TE.AM.20 - 10.TE.AM.199
VLAN100 DHCP range is now 10.0.100.126 - 10.0.100.199, 10.0.100.226 - 10.0.100.254

patfair commented:
Looks like there's a failed test that may have caught a legitimate bug.

ejordan376 commented:
Looks like you need to pass "teamPartialIp" 2 more times in network/switch.go between lines 92 and 93.

FletcherS7 commented:
If I'm understanding this right, does this look correct.

Added two new instances of teamPartialIp, removed one instance of teamPartialIp and removed one instance of switchTeamGatewayAddress.

```
                        addTeamVlansCommand += fmt.Sprintf(
			"ip dhcp excluded-address 10.%s.1 10.%s.19\n"+
			"ip dhcp excluded-address 10.%s.200 10.%s.254\n"+
				"ip dhcp pool dhcp%d\n"+
				"network 10.%s.0 255.255.255.0\n"+
				"default-router 10.%s.%d\n"+
				"lease 7\n"+
				"access-list 1%d permit ip 10.%s.0 0.0.0.255 host %s\n"+
				"access-list 1%d permit udp any eq bootpc any eq bootps\n"+
				"interface Vlan%d\nip address 10.%s.%d 255.255.255.0\n",
			teamPartialIp,
			teamPartialIp,
			teamPartialIp,
			teamPartialIp,
			vlan,
			teamPartialIp,
			switchTeamGatewayAddress,
			vlan,
			teamPartialIp,
			ServerIpAddress,
			vlan,
			vlan,
			teamPartialIp,
		        )
```
